modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,154 @@
+* <<migrate_with_external,Migrate client applications for APIs with external client Providers>>
+
+== Prerequisites
+
+Before migrating the client application, ensure that you:
+
+* Obtain the required token and credentials:
+** Client ID and client secret for the client application to migrate
+** Anypoint Platform https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/Authentication/[token]
+* Review the <<considerations,scope and limitations>> to understand the limitations and scope of the migration task
+
+[[considerations]]
+== Scope and Limitations
+
+Before you migrate your client applications to Anypoint Platform, review the following scope and limitations: 
+
+* If the organization of the API you consume has registered an external client provider, for example, PingFederate or Okta, review the section: <<external_client,Organizations with Registered External Client Providers>>.
+* Do not attempt to update the client ID and client secret of an application.
++
+This feature is not supported.
+* Do not use the same user to create more than 500 applications.
++
+The user will not be able to create applications with the same name even if the client applications in the identity provider (IdP) with the same name and different client ID and client secret.
+* Do not attempt to refresh the client ID.
++
+This feature is not unless you delete and re-register the client application.
+
+[[migrate_with_internal]]
+== Migrate Client Applications for APIs with Native Client Providers
+
+To migrate your client application:
+
+. Open a text editor and paste the following commands:
++
+[source,text,linenums]
+----
+curl --location --request POST 'https://anypoint.mulesoft.com/exchange/api/v1/organizations/:orgId/applications' \
+--header 'Authorization: Bearer <omitted>' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "clientId": <":myClientId>",
+    "clientSecret": ":myClientSecret",
+    "grantTypes": [],
+    "name": "My Application Name",
+    "redirectUri": []
+}'
+----
+. In the command snippet, replace the `orgId` parameter with your organization Id and the content of the body with the appropriate application <<properties,properties>>. 
++
+[NOTE]
+---
+You must use the organization ID of the master organization.
+---
+. Run this command at the Command prompt.
+
+[[properties]]
+== Properties Reference for the Migrate Task
+
+In the properties of the migrate command, provide values based on the following recommendations:
+
+* `clientId`: <Your client ID>
+* `clientSecret`: <Your client secret>
+* (optional)`description`: <Informative description for the applications>
++
+This property is displayed on the application page. 
+* `grantTypes`: As no IDP is provided, leave an empty array 
+* `name`: <Name of the application to be created>
++
+This property is displayed when selecting an application before requesting contracts.
```

Comment:
```suggestion
This property is displayed when selecting an application before requesting access.
```
You request access. If it is approved, a contract is created.

Reply:
Done! Thanks!